docs/source/usage/cli_commands.rst

```diff
@@ -0,0 +1,14 @@
+search
+    start in a search buffer using the querystring provided as
+    parameter. See also the SEARCH SYNTAX section of notmuch(1)
```

Comment:
On Arch Linux this is actually `notmuch-search-terms(7)`. Can any body verify this on another system? It also seems to be the case for the notmuch source distribution.